test/common/plugin/currencyPlugin.fixtures/common.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  EdgeEncodeUri,
+  EdgeParsedUri,
+  EdgeWalletInfo
+} from 'edge-core-js/types'
+
+export interface FixtureType {
+  pluginId: string
+  WALLET_TYPE: string
+  WALLET_FORMAT: string
+  'Test Currency code': string
+  key: number[]
+  xpub: string
+  'invalid key name': EdgeWalletInfo
+  'invalid wallet type': EdgeWalletInfo
+  parseUri: {
+    [testName: string]: [string, EdgeParsedUri] | [string]
+  }
+  encodeUri: {
+    [testName: string]: [EdgeEncodeUri, string] | [EdgeEncodeUri]
+  }
+  getSplittableTypes?: {
+    [walletFormat: string]: string[]
+  }
+}
```

Comment:
Remove types and merge all fixtures into one large intimidating fixture file to make it "easier" to add types.

Reply:
Although this feedback is non-negotiable, my position is against this change for these reasons:

1. Documentation for the fixtures (to make it less time-consuming for anyone to understand and add fixtures)
2. Guaranteeing correctness of the test code (no potential type-error bugs in the tests)
3. Less rigidity in refactoring tests if needed (types significantly help with refactoring especially for individuals new to the codebase)
3. Separating the fixture data by currency is an organization effort that helps to navigate the fixture data.

I'm open to having my mind changed from this position given some adequate arguments for reverting this work. So far I can only see cons: no type-safety in the test fixtures data (it will all be `any` type); no clarity of what the shape of the fixture data is so code quality will decay over time as more developers touch the test code and fixtures; there will be guidance as to what the existing test fixture data should so it'll make it very difficult to refactor or modify tests (rigidity). Increasing the probability that the test code may be incorrect because we removed a safety against a category of bugs (type-error bugs).

Reply:
I would agree w/ @samholmes . I've never liked the typeless json style of the old fixtures. It created a lot of typo issues.